### PR TITLE
feat(example): Added example

### PR DESCRIPTION
**Pull request by 9renpoto**
Added example

### CODE REVIEW

File: example/src/account/account.loader.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { Injectable } from "@nestjs/common";
+import { NestDataLoader } from "../../..";
+import { AccountService } from "./account.service";
+
+@Injectable()
+export class AccountLoader implements NestDataLoader<string, Account> {
+  constructor(private readonly accountService: AccountService) {}
+
+  generateDataLoader(): DataLoader<string, Account> {
+    return new DataLoader<string, Account>((keys) =>
```
Comment (9renpoto) [inline example/src/account/account.loader.ts:11]: not working yet 🤔

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -24,6 +24,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: yarn install --frozen-lockfile
-      - run: yarn tsc -p . --noEmit
+      - run: yarn test
```
Comment (dantehemerson) [inline .github/workflows/nodejs.yml:27]: Why you removed the compilation task? I think you should add this after the compilation task.
```suggestion
      - run: yarn tsc -p . --noEmit
      - run: yarn test
```

File: example/__tests__/app.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+          }
+        }
+      `,
+      variables: {
+        ids: ["id"],
+      },
+    });
+    expect(result.errors).toMatchInlineSnapshot(`
+      Array [
+        [GraphQLError: The loader AccountLoader is not providedError: Nest could not find AccountLoader element (this provider does not exist in the current context)],
```
Comment (9renpoto) [inline example/__tests__/app.e2e-spec.ts:47]: 🤔

File: example/src/account/account.resolver.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import { AccountLoader } from "./account.loader";
+
+@Resolver("Account")
+export class AccountResolver {
+  @Query(() => [Account])
+  public getAccounts(
+    @Args({ name: "ids", type: () => [ID] }) ids: string[],
+    @Loader(AccountLoader.name)
+    accountLoader: DataLoader<Account["id"], Account>
+  ): Promise<(Account | Error)[]> {
```
Comment (9renpoto) [inline example/src/account/account.resolver.ts:14]: return `: Promse<(Account | Error>[]` ? 

https://github.com/krislefeber/nestjs-dataloader/blob/master/README.md#using-the-nestdataloader

File: index.ts [hunk 5]
```diff
@@ -64,12 +68,15 @@ export class DataLoaderInterceptor implements NestInterceptor {
 /**
  * The decorator to be used within your graphql method.
  */
-export const Loader = createParamDecorator(async (data: string, [_, __, ctx]) => {
-  if (ctx[NEST_LOADER_CONTEXT_KEY] === undefined) {
-    throw new InternalServerErrorException(`
+export const Loader = createParamDecorator(
+  async (data: unknown, context: ExecutionContext) => {
+    const ctx = GqlExecutionContext.create(context).getContext();
```
Comment (9renpoto) [inline index.ts:73]: deprecated #24

File: .github/workflows/nodejs.yml [hunk 6]
```diff
@@ -25,5 +25,10 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: yarn install --frozen-lockfile
       - run: yarn tsc -p . --noEmit
+      - run: yarn test
         env:
           CI: true
+      - name: Coveralls
+        uses: coverallsapp/github-action@master
```
Comment (9renpoto) [inline .github/workflows/nodejs.yml:32]: Hi @krislefeber , can you activate coveralls ? :)

Comment (9renpoto) [reply .github/workflows/nodejs.yml:32]: https://coveralls.io/github/krislefeber/nestjs-dataloader

**Comment by 9renpoto** [comment]
🙏

**Pull request merged**